lib/fastlane/helper/plugin_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Fastlane
+  module Actions
+    # This method will import a plugin
+    def self.plugin(gem)
+      Kernel.require "#{gem}"
```

Comment:
This method shouldn't be needed now we're using CLAide's system, I'll remove this.